app/views/bulk_updates/index.html.erb

```diff
@@ -0,0 +1,28 @@
+<div class="row row-content">
+  <section>
+    <h2>Bulk Updates for <%= @provider_id %></h2>
+
+    <table class="bulk-updates-list-table">
+      <thead>
+        <tr>
+          <th>Task ID</th>
+          <th>Status</th>
+        </tr>
+      </thead>
+      <tbody>
+        <% if @tasks.blank? %>
+          <tr>
+            <td colspan="2">No tasks found</td>
+          </tr>
+        <% else %>
+          <% @tasks.each do |task| %>
+            <tr>
+              <td><%= link_to task.fetch('task-id', nil), bulk_update_path(task.fetch('task-id', 'not found')) %></td>
```

Comment:
You could use `link_to_if` here, if there is no task-id then you're currently displaying a link with no text and supplying 'not found' as the ID. I'd prefer this not be a link at all, *or better yet, if no task-id exists, this row shouldn't be displayed because its a useless piece of data*